hypertrace-view-generator/helm-test/templates/all-views-creation.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: batch/v1
```

Comment:
Can we test this onec as all-views-creator uses - org.hypertrace.core.viewcreator.pinot.PinotMultiTableCreationTool?

@JBAhire / @rish691  Is it used in `hypertrace.sh` install script?